tests/scripts/thread-cert/node_cli.py

```diff
@@ -77,11 +81,16 @@ def __init_sim(self, nodeid, mode):
 
     def __init_ncp_sim(self, nodeid, mode):
         """ Initialize an NCP simulation node. """
+        if 'NCP_FILE' in os.environ:
+            args = ' %s' % os.environ['NCP_FILE']
+        else:
+            args = ''
+
         if "top_builddir" in os.environ.keys():
             builddir = os.environ['top_builddir']
-            cmd = 'spinel-cli.py -p %s/examples/apps/ncp/ot-ncp-%s -n' % (builddir, mode)
+            cmd = 'spinel-cli.py -p "%s/examples/apps/ncp/ot-ncp-%s%s" -n' % (builddir, mode, args)
```

Comment:
That is great and what we really need. I have just one question, what do you think about creating spinel-radio-cli.py to test only link_raw functionality of ncp-radio device, in the future?

Reply:
Yes, I think so. We could create a new test script to test radio only or link raw functionality in the future. This script could also be used as a simple utility to verify radio only firmware.